tokens/src/themes/light/components/Avatar.json

```diff
@@ -1,7 +1,8 @@
 {
+  "$type": "color",
   "color": {
```

Comment:
[curious] Based on the [migration guidelines](https://styledictionary.com/version-4/migration/#cti-reliance):

<img width="917" alt="image" src="https://github.com/user-attachments/assets/eec5cdba-4899-4a27-9787-433cdc8a7faa">

It _seems_ the top-level `"color"` key here isn't actually needed anymore for `style-dictionary` to determine the CTI (Category-Type-Item), given the `$type` annotation. Is this something that was explored? It might help simplify JSON files slightly by removing possible layer(s) of nesting. If it is possible to nix the extra nesting layer, curious to hear your thoughts around scope in tackling that broadly.

Reply:
Note: I might be misinterpreting this, where the top-level `color` is still needed from a grouping perspective; just not from a type pov.